PrebidMobile/ConfigurationAndTargeting/Targeting.swift

```diff
@@ -255,6 +257,45 @@ public class Targeting: NSObject {
         return transformedUserIdArray.isEmpty ? nil : transformedUserIdArray
     }
     
+    // MARK: - SharedId
+    
+    public var sharedIdEnabled: Bool = false
+    
+    public var sharedId: ExternalUserId? {
+        let sharedIdAllowed: Bool = {
```

Comment:
Please encapsulate the code for managing  SharedId in a standalone class. The Targeting class shoud work only as a facade.